ui.tests/test-module/specs/formContainer/formContainer.runtime.cy.js

```diff
@@ -0,0 +1,111 @@
+
+
+
+describe("Form Runtime with Fragment", () => {
+const pagePath = "content/forms/af/core-components-it/samples/formcontainer/basic.html"
+
+const selectors = {
+    hamburgerIcon : `.cmp-adaptiveform-container__hamburger-menu-icon`,
```

Comment:
Can we refer the selectors from the class directly, rather than copy pasting it here ? We have already exposed these as constants from the class

Reply:
Can we import that js class here its in another maven project? I need to check that.

Reply:
Done! Imported from class itself